environment.yml

```diff
@@ -0,0 +1,27 @@
+name: biovectors
+channels:
+ - conda-forge
```

Comment:
As a non-conda expert, just curious if you need this channel since you're not using it here.

Reply:
This is the default channel to get initial dependencies. I left it in there as I don't want things to go awry, but I understand your point. Will default to leaving it in for now.